### PR TITLE
Fix get_transaction_by_txid of cfx

### PR DESCRIPTION
**Pull request by huazhouwang**
## What does this implement/fix? Explain your changes.
状态判断有误

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://onekeyhq.atlassian.net/browse/OK-2806

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

**Review by liyanhrxy — APPROVED** [review]

File: electrum_gui/common/provider/chains/cfx/conflux.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def get_transaction_by_txid(self, txid: str) -> Transaction:
             )
             status = (
                 TransactionStatus.CONFIRM_SUCCESS
-                if receipt.get("outcomeStatus") == "0x1"
+                if receipt.get("outcomeStatus") == "0x0"
```
Comment (taimanhui) [inline electrum_gui/common/provider/chains/cfx/conflux.py:126]: 看它[文档](https://conflux-chain.github.io/conflux-doc/json-rpc/#example_20)，这里是个数字呢

**Review by taimanhui — COMMENTED** [review]
莫名其妙，文档里也没说这个返回值是啥，只有example里是0...

File: electrum_gui/common/provider/chains/cfx/conflux.py (hunk 1 above)
Comment (huazhouwang) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: @liyanhrxy

Comment (liyanhrxy) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: @taimanhui 说明[这里](https://developer.confluxnetwork.org/conflux-doc/docs/json_rpc#cfx_gettransactionreceipt)有

Comment (taimanhui) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: 这里判断错了，是个数字不是个字符串啊...文档和示例都是数字

Comment (huazhouwang) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: 目前我测试来看 outcomeStatus 确实在交易成功时返回 "0x0" 字符串

Comment (huazhouwang) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: 目前我只校验了成功的交易，有失败的交易给我校验下吗？我通过 https://confluxscan.io/ 没找到失败的交易

Comment (taimanhui) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: @huazhouwang 成功了就行吧？难道不成功也返回"0x0" 😂

**Review by taimanhui — APPROVED** [review]

**Comment by taimanhui** [comment]
这个合得快不用分支呢，直接推这个新的就好，不包含其它的干扰项

**Comment by huazhouwang** [comment]
好

File: electrum_gui/common/provider/chains/cfx/conflux.py (hunk 1 above)
Comment (liyanhrxy) [reply electrum_gui/common/provider/chains/cfx/conflux.py:126]: 成功是“0x0”, 失败是“0x1”(由`cfx_getTransactionByHash` 的`status`猜测)

**Pull request merged**